HarmonyCore/FileIO/FileChannelManager.dbl

```diff
@@ -170,37 +176,93 @@ namespace Harmony.Core.FileIO
         private class LoadStats
             private lid, int
             private totalConnections, int
-            private totalFiles, int
-			
+            private openConStartTime, @Dictionary<int, DateTime>, new Dictionary<int, DateTime>()
+            private openChStartTime, @Dictionary<int, DateTime>, new Dictionary<int, DateTime>()
+
             public method LoadStats
                 lid, int
             proc
                 this.lid = lid
                 this.totalConnections = 0
-                this.totalFiles = 0
             endmethod
 
             public method AddConnection, void
             proc
                 this.totalConnections += 1
             endmethod
 
-            public method AddFile, void
+            public method GetTotalConnections, int
+            proc
+                mreturn this.totalConnections
+            endmethod
+
+            public method RecordConTime, void
+                hand, int
             proc
-                this.totalFiles += 1
+                data startTime = new DateTime()
+                startTime = DateTime.Now
+                openConStartTime.Add(hand, startTime)
             endmethod
 
-            public method GetTotalConnections, int
+            public method GetConnectionTime, @TimeSpan
             proc
-                mreturn this.totalConnections
+                data currentTime = new DateTime()
+                currentTime = DateTime.Now
+                data activeConnectionsTime, @TimeSpan, new TimeSpan()
+                data interval, @TimeSpan, new TimeSpan()
+                data kvp, KeyValuePair<int, DateTime>
+                foreach kvp in openConStartTime
+                begin
+                    interval = currentTime - kvp.Value
+                    activeConnectionsTime = activeConnectionsTime.Add(interval)
+                end
+
+                mreturn activeConnectionsTime
             endmethod
 
-            public method GetTotalFiles, int
+            public method RecordChTime, void
```

Comment:
My preference for naming public methods is to not use shorteners like 'Ch' for 'Channel'. Can you expand these shortened names?